include/mfmg/dealii/lanczos.hpp

```diff
@@ -47,8 +47,8 @@ class Lanczos
                         VectorType const &initial_guess);
 
   static std::tuple<std::vector<double>, std::vector<double>>
-  details_calc_tridiag_epairs(std::vector<double> const &t_maindiag,
-                              std::vector<double> const &t_offdiag,
+  details_calc_tridiag_epairs(std::vector<double> const &main_diagonal,
+                              std::vector<double> const &sub_diagonal,
```

Comment:
As much as I did not like the variable names before, this change introduces a lot of noise in the diff and distracts from the changes that matter.  Please avoid in the future.   Consider submitting a separate PR with the renaming which would probably be immediately approved and merged.  Do not change this PR, this just an observation.

Reply:
Noted.